HUD/IsMouseCursorAboveInstructionalButtons.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: HUD
+aliases: ["0x3D9ACB1EB139E702"]
+---
+## _IS_MOUSE_CURSOR_ABOVE_INSTRUCTIONAL_BUTTONS
+
+```c
+// 0x3D9ACB1EB139E702
+BOOL _IS_MOUSE_CURSOR_ABOVE_INSTRUCTIONAL_BUTTONS();
```

Comment:
When changing return types or parameter types, be sure to add `cs_type(int)` for C# to keep backwards compatibility.

Reply:
should be fixed in the new commit :)